### PR TITLE
Added FollowService , ReadmeService and GitIgnoreService

### PR DESCRIPTION
**Pull request by Javohir-Khursanboyev**

### CODE REVIEW

File: CodeHub.Service/Services/FollowService.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        this.repository = repository;
+        this.userService = userService;
+    }
+
+    public async Task<FollowViewModel> CreateAsync(FollowCreateModel follow)
+    {
+        if (follow.FollowerId == follow.FollowingId)
+            throw new CustomException(400, "You can't follow yourself");
+
+        var existFollower = await repository.SelectByIdAsync(follow.FollowerId)
```
Comment (makhammadsoliyev) [inline CodeHub.Service/Services/FollowService.cs]: Did you need to use userService instead of repository which is of follow type?

**Review by makhammadsoliyev — CHANGES_REQUESTED** [review]
You should use userService to check user/follow instead of follow repository

File: CodeHub.Service/Services/FollowService.cs (hunk 1 above)
Comment (Javohir-Khursanboyev) [reply CodeHub.Service/Services/FollowService.cs]: you are right

**Review by makhammadsoliyev — APPROVED** [review]
Not Bad

**Pull request merged**